randomtest.js

```diff
@@ -458,6 +458,12 @@ Scene.prototype.choice = function choice(data, fakeChoice) {
       };
       this.temps.fakeChoiceLines = fakeChoiceLines;
     }
+    if (!this.temps.__choiceEnds) {

```

Comment:
Okay.  Is that the only change?  Can we merge now and update cosmetics like this with a second pull request, or do you want this changed before the merge?